stingray/covariancespectrum.py

```diff
@@ -206,14 +206,7 @@ def _construct_energy_covar(self):
 
             self.energy_covar[energy] = covar
 
-            # Calculate error
-            if self.error is None:
-                std = np.mean(lc_ref)**0.5
-            elif isinstance(self.error, collections.Iterable):
-                std = np.mean(self.error)  # Iterable of numbers
-            else:  # Single float number
-                std = self.error
-
```

Comment:
Can we perhaps rename this `self.std` or something? I'm worried that people will mistake this quantity for the error on the covariance.